app-misc/vcontrold/vcontrold-0.98.5.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Daemon for communication with Viessmann Vito heatings"
+HOMEPAGE="https://github.com/openv/vcontrold/"
+SRC_URI="https://github.com/openv/vcontrold/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+man +vclient vsim"
+
+RDEPEND="dev-libs/libxml2"
+DEPEND="${RDEPEND}
+	man? ( dev-python/docutils )"
+
+src_prepare() {
+	sed "s/@VERSION@/${PV}/" "src/version.h.in" \
+		> "src/version.h" || die "Setting version failed"
+
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DMANPAGES="$(usex man)"
+		-DVCLIENT="$(usex vclient)"
+		-DVSIM="$(usex vsim)"
+	)
+
+	cmake-utils_src_configure
+}
+
+src_install() {
+	cmake-utils_src_install
+	doinitd "${FILESDIR}/vcontrold"
+	dodir /etc/vcontrold
```

Comment:
No need for `dodir` before `insinto`.

Reply:
Changed with ff99f8e051a83dc329fcc55a162871b0d84fd0e7